### PR TITLE
fixed broken link to app-development documentation

### PR DESCRIPTION
**Pull request by venkatkalluru**
@enothereska @mjsax please review

P.S: I randomly selected two of the reviewers looking at other PRs.

### CODE REVIEW

**Comment by venkatkalluru** [comment]
[clabot:check]

File: kafka-clients/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 This directory includes projects demonstrating how to use the Java Kafka producer
 and consumer. You can find detailed explanation of the code at the
-[application development section](http://confluent.io/docs/2.0.1/app-development.html)
+[application development section](http://docs.confluent.io/3.2.0/app-development.html)
```
Comment (miguno) [inline kafka-clients/README.md]: This should be `/3.2.1/` rather than `/3.2.0/`.

Comment (venkatkalluru) [inline kafka-clients/README.md]: @miguno updated the version, please review

**Comment by miguno** [comment]
Many thanks @venkatkalluru !

**Pull request merged**